bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -133,7 +143,7 @@ spec:
           fi
           exec /usr/share/openvswitch/scripts/ovn-ctl run_sb_ovsdb \
             --no-monitor \
-            --ovn-nb-log="-vconsole:${OVN_LOG_LEVEL} -vfile:off"
+            --ovn-sb-log="-vconsole:${OVN_LOG_LEVEL} -vfile:off"
```

Comment:
Can you make this a separate commit? (eg, if we ended up having to revert the SSL stuff there's no reason to revert this too)

Reply:
dine